tinygrad/runtime/autogen/comgr.py

```diff
@@ -27,9 +28,12 @@ def char_pointer_cast(string, encoding='utf-8'):
     return ctypes.cast(string, ctypes.POINTER(ctypes.c_char))
 
 
-
 _libraries = {}
-_libraries['libamd_comgr.so'] = ctypes.CDLL(os.getenv('ROCM_PATH', '/opt/rocm/')+'/lib/libamd_comgr.so')
+_libraries['libamd_comgr.so'] = ctypes.CDLL(
```

Comment:
Don't change autogenerated files. See `autogen_stubs.sh`

Reply:
thanks, yeah, I found it as well.